db/controllers/reviews.controllers.js

```diff
@@ -8,3 +11,20 @@ exports.getReviewById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.incrementVotes = (req, res, next) => {
+  const {
+    params: { review_id: id },
+    body: { inc_votes: votes },
+  } = req;
+  // console.log(typeof votes, ":", votes, "<<<votes");
+  if (!votes) next({ status: 404, msg: "Passed malformed body" });
```

Comment:
I feel like the controller's job should be to deal with the request and response. It is the models job to handle the data that has been sent in with on the request and perform the requests to the database with that data.

I would move these errors to the model. Also regarding the checking of the typeof the number, this error will be handled by PSQL when you make the request.